Vagrantfile

```diff
@@ -20,16 +20,22 @@ DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
 CODENAME=$(lsb_release -cs)
 echo "deb http://repos.mesosphere.io/${DISTRO} ${CODENAME} main" | sudo tee /etc/apt/sources.list.d/mesosphere.list
 
-apt-get -q -y update
+apt-get -o Acquire::http::Timeout=1 -o Acquire::ftp::Timeout=1 -y update
 echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
-apt-get -q -y install oracle-java8-installer
-apt-get -q -y install oracle-java8-set-default
-apt-get -q -y install libcurl3
-apt-get -q -y install zookeeperd
-apt-get -q -y install aria2
```

Comment:
@erikdw There aren't so many error messages when the package is aready installed. Following is what is displayed when oracle-java8-installer is already installed

```
==> master: Reading package lists...
==> master: Installing oracle-java8-installer...
==> master: Installing oracle-java8-installer
==> master: Reading package lists...
==> master: Building dependency tree...
==> master: Reading state information...
==> master: oracle-java8-installer is already the newest version.
==> master: 0 upgraded, 0 newly installed, 0 to remove and 31 not upgraded.
==> master: Installing oracle-java8-set-default...
```